time stamp
object_detection/ObjectWithType[] objects
